base/pkg/git.jl

```diff
@@ -58,6 +58,10 @@ attached(; dir="") = success(`symbolic-ref -q HEAD`, dir=dir)
 branch(; dir="") = readchomp(`rev-parse --symbolic-full-name --abbrev-ref HEAD`, dir=dir)
 head(; dir="") = readchomp(`rev-parse HEAD`, dir=dir)
 
+function iscommit(sha1s::Array{ASCIIString}; dir="")
+    indexin(sha1s,split(readchomp(`log --all --format=%H`, dir=dir),"\n")).!=0
```

Comment:
Can you check whether the latest versions of git, 2.3, 2.4 etc still accept `--format` ? The docs for recent versions are a bit ambiguous.